app/assets/stylesheets/application.css

```diff
@@ -149,3 +149,17 @@ position: absolute;
   box-shadow: none !important;
   text-align: center;
 
+<<<<<<< Updated upstream
```

Comment:
If I'm reading this right, there seems to be remnants of a **merge conflict** here.